5._Governance.md

```diff
@@ -6,57 +6,65 @@ As a community effort organized within the [Supply Chain Integrity WG](https://g
 
 <!-- Note: The above text was copied with modification from https://github.com/hyperledger/fabric/blob/main/docs/source/CONTRIBUTING.rst -->
 
-**1.4. Steering Committee Members.** The "Steering Committee" is the body that is responsible for overseeing the overall activities of the Project. The Steering Committee consists of up to 7 Participants (each, a "Steering Committee Member") and will initially consist of the Steering Committee Members so designated as of the date of initial adoption of this SLSA Governance Policy. The Steering Committee will meet regularly as needed, but no less then once per quarter. Examples of the responsibilities of the Steering Committee include:
+**1.4. Steering Committee Members.** The "Steering Committee" is the body that is responsible for overseeing the overall activities of the Project. The Steering Committee consists of 5 Participants (each, a "Steering Committee Member") and will initially consist of the Steering Committee Members so designated as of the date of initial adoption of this SLSA Governance Policy. The Steering Committee will meet regularly as needed, but no less then once per quarter. Examples of the responsibilities of the Steering Committee include:
 
 * enabling the smooth running of the Project
-* coordinating activities between workstreams and Maintainers
-* collectively reviewing and revising the roadmap on a biannual basis
+* active participation within the community
+* collectively reviewing and revising the Project mission, vision, strategy, and roadmap every 6 months or more frequently if needed, and publishing Steering Committee meetings notes
+* publishing quarterly updates suitable for OpenSSF TAC and other working groups' consumption
 * participating in strategic planning, such as coordinating face-to-face meetings or suggesting and approving changes to the governance model
-* creating or restructuring workstreams
+* managing the lifecycle of existing and proposed Workstreams within the Project with the consensus input from the community of Participants
+* creating or restructuring Workstreams
 * responding to specific issues or concerns above and beyond the domain of the various workstreams
 * making decisions when community consensus cannot be reached, pursuant to the appeal process documented below
+* holding Maintainers accountable to the standards of what makes SLSA an effective Project and representative of a diverse community of Participants and Contributors. In the extreme event that a majority of existing Maintainers are unavailable to approve additions or removals for an extended period of time, the Steering Committee may add or remove Maintainers to bring the Project back to a functioning state
+* administration of slsa-framework GitHub organization, including maintaining team memberships for Maintainers, setting security defaults such as branch protection for the organization, managing secrets, etc.
```

Comment:
```suggestion
* administration of slsa-framework GitHub organization, including maintaining team memberships for Maintainers, setting security defaults such as branch protection for the organization, managing secrets, etc.

The Steering Committee will be comprised of Members from a diverse set of affiliations. An affiliation must not be represented more than once on the Steering Committee. If affiliation changes during a Steering Committee Member's term which would cause overrepresentation, a Steering Committee Member from that affiliation must resign.
```